CondFormats/PPSObjects/interface/TotemDAQMapping.h

```diff
@@ -51,6 +52,7 @@ class TotemDAQMapping {
   std::map<uint8_t, TotemTimingPlaneChannelPair> totemTimingChannelMap;
 
   void insert(const TotemFramePosition& fp, const TotemVFATInfo& vi);
+  void insert(const TotemT2FramePosition& fp2, const TotemVFATInfo& vi);
```

Comment:
Do I understand that correctly that this is changing a member of this class? CondFormat member states cannot change when the CondFormat is in the DB, well, in your case, I guess this is going to be fine since the object is not connected to any record in the DB. Is that correct?

Reply:
Adding this new member function, you mean? It seems like this class is not in the DB since it's not serializable, yes.

Reply:
ok feel free to have this thread resolved